sass/typography/_functions.scss

```diff
@@ -51,6 +51,7 @@
 
 /// Produces a type style map of values that adheres to the ITypeStyle map.
 /// @access public
+/// @group Typography
```

Comment:
This can be defined at the top of the document, like this:

```scss
////
/// @group elevations
////
```

In this way, you won't have to add it to all mixins/functions.